pkg/webhook/mutation/pod/v2/webhook.go

```diff
@@ -52,7 +53,11 @@ func NewInjector(kubeClient client.Client, apiReader client.Reader, metaClient c
 func (wh *Injector) Handle(_ context.Context, mutationRequest *dtwebhook.MutationRequest) error {
 	wh.recorder.Setup(mutationRequest)
 
-	if !wh.isInputSecretPresent(mutationRequest) {
+	if !wh.isInputSecretPresent(mutationRequest, consts.BootstrapperInitSecretName) {
+		return nil
+	}
+
+	if !wh.isInputSecretPresent(mutationRequest, consts.BootstrapperInitCertsSecretName) {
```

Comment:
this should more complex checks:
- for the `BootstrapperInitSecretName` it can stay as is, because it should always be there if we inject
- for the `BootstrapperInitCertsSecretName` it should only be there if certs are used:
   - by default (if you have an AG) there will be the autogenerated certs, but this can be turned off
   - if you have no AG + no certs defined, you shouldn't have the secret there, so you shouldn't demand it

Reply:
I fixed it in 9eee9d599